DataFormats/ProtonReco/interface/ProtonTrackExtra.h

```diff
@@ -46,6 +46,8 @@ namespace reco
       /// List of RP tracks that contributed to this global track
       const RPList& contributingRPs() const { return contributing_rp_ids_; }
 
+      // TODO: replace contributing RPs with contributing local lite tracks ??
```

Comment:
Up to you! It is true that one overlaps the other through [this getter](https://github.com/cms-sw/cmssw/blob/master/DataFormats/CTPPSReco/interface/CTPPSLocalTrackLite.h#L29-L33)...